clvm/SExp.py

```diff
@@ -61,6 +61,8 @@ def _to_sexp_type(
             return NULL
         if v == []:
             return NULL
+        if isinstance(v, str):
```

Comment:
This looks really handy. Two small things I worry about with this:

* Mechanism / comment to warn about using this from consensus/blockchain code (if someone adds a CLVM operator in the future, warn them away from this, because it goes through Python's string conversion)
* UTF-8 conversion surprise

Reply:
I can't decide if we should quietly support strings or if we should raise an error. The downside of supporting strings is it lets users be sloppy with `bytes` vs `str`. The upside is it lets users be sloppy with `bytes` vs `str`. Maybe that's okay, since the main use case for `to` is for use with quick-hack slop code.